File: test/fixtures/node/post_with_quotes_anywhere.js
import fetch from 'node-fetch';

fetch('http://localhost:28139', {
  method: 'POST',
  headers: {
    'A': "''a'",
    'B': '"',
    'Cookie': 'x=1\'; y=2"',
    'Content-Type': 'application/x-www-form-urlencoded',
    'Authorization': 'Basic ' + btoa('ol\':asd"')
  },
  body: 'a=b&c="&d=\''
});

File: test/fixtures/clojure/post_with_quotes_anywhere.clj
(require '[clj-http.client :as client])

(client/post "http://localhost:28139" {:headers {"A" "''a'",
                                                 "B" "\"",
                                                 "Cookie" "x=1'; y=2\"",
                                                 "Content-Type" "application/x-www-form-urlencoded"}
                                       :basic-auth ["ol'" "asd\""]
                                       :body "a=b&c=\"&d='"})
